src/css/stake-page.scss

```diff
@@ -0,0 +1,255 @@
+#home-stake  {
```

Comment:
@astanovferuz, all of the styles for this stake page should be inside a parent block `.build-content` like as following:
```
.build-content {
...
}
```